ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -36,7 +36,7 @@
 public class Command {
 
   @VisibleForTesting
-  public static final int VERSION = 6;
+  public static final int VERSION = 7;
```

Comment:
How is this related to 0.18?

Reply:
This is used to make sure that old servers don't run new commands. The 0.18 branch is on version 6, so we want master to be on a later version.